From f557f7487d9aae0c901a740b9a446568677b8bb3 Mon Sep 17 00:00:00 2001
From: Jan Cholasta <jcholast@redhat.com>
Date: Mon, 25 Jul 2016 15:58:20 +0200
Subject: [PATCH] compat: fix ping call

Copy & paste accident caused the ping command to be called with an unwanted
argument, which results in an exception.

Remove the argument to fix it.

https://fedorahosted.org/freeipa/ticket/6129
---
 ipaclient/remote_plugins/compat.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ipaclient/remote_plugins/compat.py b/ipaclient/remote_plugins/compat.py
index 40521af450aafca83f33d1723b4fd9e27ef8d96f..aef5718fcaade157487c0e65562c3bc8a11ad7de 100644
--- a/ipaclient/remote_plugins/compat.py
+++ b/ipaclient/remote_plugins/compat.py
@@ -39,7 +39,7 @@ def get_package(api, client):
     try:
         server_version = env['result']['api_version']
     except KeyError:
-        ping = client.forward(u'ping', u'api_version', version=u'2.0')
+        ping = client.forward(u'ping', version=u'2.0')
         try:
             match = re.search(u'API version (2\.[0-9]+)', ping['summary'])
         except KeyError:
-- 
2.7.4

